docs/intro.rst

```diff
@@ -11,7 +11,7 @@ leading ultra-high performance, distributed networking databus.
 
 *RTI Connector* is a family of simplified APIs for publishing and subscribing
 to the *Connext DDS* Databus in programming languages such as Python and JavaScript. 
-(RTI also offers a more comprehensive `Python API <https://community.rti.com/static/documentation/connext-dds/current/api/connext_dds/api_python/intro.html>`__, 
+(RTI also offers a more comprehensive `Python API <https://community.rti.com/static/documentation/connext-dds/current/doc/api/connext_dds/api_python/intro.html>`__, 
```

Comment:
I think we should link to the GitHub repo instead https://github.com/rticommunity/connextdds-py

The documentation is currently built on a best-effort basis, and doesn't necessarily follow our release timeline.

Reply:
I changed it. 

Once you're at https://github.com/rticommunity/connextdds-py, the link to the Python API Reference goes to a 6.0.1 URL: https://community.rti.com/static/documentation/connext-dds/6.0.1/api/connext_dds/api_python/index.html. I'm assuming that's OK.